### ISSUE TITLE
Update new contributor experience

### ISSUE THREAD

**Issue by jacobweinstock** [tinkerbell/tink#417]
Getting started with the Tinkerbell stack and through this Tink repo the local development path, for me, had a few hurdles. 

- I found it difficult to understand and spin up Tink without diving deep into the code.
- I haven't been able to generate TLS certificates locally using the scripts found in this repo. They were all very coupled to vagrant with virtualbox.
- There were a lot of environment variables that were difficult to understand what to set them to without, again, diving deep into the code.
- `make run` did not work for me.
```bash
ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
```
- Documentation was difficult to find

## Expected Behaviour
lower barrier to use, contribute and understand.

## Current Behaviour
see above

## Possible Solution
I have a PR here: https://github.com/tinkerbell/tink/pull/418 with proposed changes to add an environment variable to disable TLS in the server and cli. Documentation for all environment variables. New top-level docker compose file. And updated `make run` target. See PR 

## Your Environment

* Operating System and version:
MacOS Catalina

* How are you running Tinkerbell? Using Vagrant & VirtualBox, Vagrant & Libvirt, on Packet using Terraform, or give details:
Running Tink locally with docker-compose.

* Link to your project or a code example to reproduce issue:

**Comment by tstromberg** [comment]
@jacobweinstock - IMHO, this issue would be a lot more actionable if split into multiple issues. Would you mind doing so, so that they can be attacked one at a time?

**Comment by jacobweinstock** [comment]
I feel that all of these except one have actually been resolved.

_I found it difficult to understand and spin up Tink without diving deep into the code._

- `make run` now easily stands up the Tink stack.

_I haven't been able to generate TLS certificates locally using the scripts found in this repo. They were all very coupled to vagrant with virtualbox._

- `make run` and the docker-compose.yaml now handle TLS certs more clearly.

_There were a lot of environment variables that were difficult to understand what to set them to without, again, diving deep into the code._

- This is the one I feel is still unresolved. I have a closed PR (#418) that documents the environment variables. I will reopen it.

_`make run` did not work for me._

- This is working well for me now.

**Comment by nshalman** [comment]
#545 is a very good start. If anyone wants to pick up from there and complete the work it would be much appreciated.
